### ISSUE TITLE
Submenu custom name does not work

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/handsontable#3550]
Demo: http://jsfiddle.net/xryogmqd/
Reported at: https://forum.handsontable.com/t/translating-context-menu-sub-entries-breaks-functionality/389

This menu represents `borders` menu when I pass only `borders: top` option and change it's name. Name is changed but the function does not want to work.
It works as expected when we're not accessing submenu: http://jsfiddle.net/0kqh6psg/

**Comment by Gronan** [comment]
any update on this issue ?

**Comment by AMBudnik** [comment]
No @Gronan not yet. I will check what can be done and what may be causing it and if I'll find any useful piece of information I'll be back with an update.

**Comment by AMBudnik** [comment]
Still does not work in v0.35.1. Updated demo http://jsfiddle.net/9a9ez3h2/

**Comment by nicolas-schreiber** [comment]
Any update on this? Still seems to not be working in v0.38.1

**Comment by AMBudnik** [comment]
Sorry @nicolas-schreiber 

I do not have any updates on that issue but checked this example http://jsfiddle.net/handsoncode/uea06stv/ and it seems to work well. Do you have any example when it doesn't?

**Comment by matejsvajger** [comment]
@AMBudnik what @nicolas-schreiber meant is that the original functionality of the menu item is gone when you change the `name`. You implemented a custom callback.

**Comment by AMBudnik** [comment]
@matejsvajger 

you're right, my bad. I've used a single-level menu. Here's the old one (with submenu) for the latest version 2.0.0 that doesn't work as expected http://jsfiddle.net/handsoncode/u31tshu1/

**Comment by AMBudnik** [comment]
The issue is replicable in v `2.0.0.` and doesn't seem to work fine in any of the previous versions.

**Comment by pnowak** [comment]
This issue is still replicable using v8.0.0. Here's an updated demo - https://jsfiddle.net/cbkLqta1/

**Comment by giorgitus** [comment]
Any update?

**Comment by AMBudnik** [comment]
Hi @giorgitus the issue hasn't been added to a milestone yet. I will make sure to inform you and other users listed above after fixing this issue.

**Comment by AMBudnik** [comment]
Hi @matejsvajger @Gronan @giorgitus @nicolas-schreiber 

I have good news. It seems that the issue is no longer replicable using v15.3.0. Here's an updated demo: https://jsfiddle.net/8cjv5xnh/1/ 

If you have any issues with the latest version, please let me know at support@handsontable.com
